components/launcher-client/src/client.rs

```diff
@@ -21,21 +21,24 @@ use ipc_channel::ipc::{IpcOneShotServer,
                        IpcReceiver,
                        IpcSender};
 use std::{collections::HashMap,
-          fs,
           io,
           path::Path};
 
 type Env = HashMap<String, String>;
 type IpcServer = IpcOneShotServer<Vec<u8>>;
 
 pub struct LauncherCli {
-    tx:   IpcSender<Vec<u8>>,
-    rx:   IpcReceiver<Vec<u8>>,
+    tx: IpcSender<Vec<u8>>,
+    rx: IpcReceiver<Vec<u8>>,
+    #[cfg_attr(windows, allow(dead_code))]
     pipe: String,
 }
 
+#[cfg(not(windows))]
 impl Drop for LauncherCli {
     fn drop(&mut self) {
+        use std::fs;
```

Comment:
You could save a few lines and omit the `use` statement with `std::fs::remove_file`